common-java-lib-package/src/main/resources/manifest.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: java-common-lib
+# Component identifier. This identifier matches artifact path in Zowe Artifactory https://zowe.jfrog.io/.
+id: org.zowe.apiml.sdk.java-common-lib-package
+# Component version is defined in gradle.properties for Gradle project
+# Human readable component name
+title: Shared directory containing the common native libraries
+# Human readable component description
+description: Directory that contains the API ML native libraries.
+license: EPL-2.0
+repository:
+  type: git
+  url: https://github.com/zowe/api-layer.git
```

Comment:
Shouldn't be `https://github.com/zowe/common-java.git`?

Reply:
good catch!